rero_ils/modules/items/api/circulation.py

```diff
@@ -1150,7 +1150,7 @@ def status_update(
             if item['status'] != ItemStatus.MISSING and on_shelf is True:
                 item['status'] = ItemStatus.ON_SHELF
         if dbcommit:
-            item.commit()
+            # item.commit()
```

Comment:
why? You can remove this comment.

Reply:
Because it is done with `item.dbcommit(reindex=True, forceindex=True)`